services/httpoverrpc/httpoverrpc.proto

```diff
@@ -22,14 +22,18 @@ package HTTPOverRPC;
 
 // The HTTPOverRPC service definition
 service HTTPOverRPC {
-  // Make an HTTP call to localhost
-  rpc Localhost(LocalhostHTTPRequest) returns (HTTPReply) {}
+  // Make an HTTP call to specified host
+  rpc Host(HostHTTPRequest) returns (HTTPReply) {}
```

Comment:
This change is slightly dangerous. Renaming the RPC method name in-place breaks backwards compatibility because the method name's passed as a string over the grpc connection to decide which method to call.

However, I think it's okay because the command's been around for less than a month (https://github.com/Snowflake-Labs/sansshell/pull/266) and it's unlikely that anyone's picked up a strong dependency on that at this time. Previous behavior can be achieved on a policy level with

```
allow {
  input.message.name == "localhost"
  input.message.protocol == "http"
}
```

cc @sfc-gh-jallie because he helped review the original PR.

Reply:
I made the change in place also because this action was release very recently, but the best way is to create a new rpc method and deprecate the old one gradually, right? So here should I keep the Localhost rpc method or continue with the current approach.

Reply:
I'm happy with the current approach.

If the feature were around for more than 6-12 months, we could mark it as deprecated but we'd want to keep it around indefinitely for backwards compatibility.